sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/CatalystQlSuite.scala

```diff
@@ -201,4 +201,12 @@ class CatalystQlSuite extends PlanTest {
     parser.parsePlan("select sum(product + 1) over (partition by (product + (1)) order by 2) " +
       "from windowData")
   }
+
+  test("subquery") {
+    parser.parsePlan("select (select max(b) from s) ss from t")
```

Comment:
Since plan checking is too easy to break, I added test for plan, finally remove them.